packages/react-native-codegen/src/parsers/parsers-commons.js

```diff
@@ -107,6 +106,43 @@ function assertGenericTypeAnnotationHasExactlyOneTypeParameter(
   }
 }
 
+function buildSchema(
+  contents: string,
+  filename: ?string,
+  parser: Parser,
+  Visitor: ({isComponent: boolean, isModule: boolean}) => {
+    [type: string]: (node: $FlowFixMe) => void,
+  },
+  wrapComponentSchema: (config: ComponentSchemaBuilderConfig) => SchemaType,
+  buildComponentSchema: (ast: $FlowFixMe) => ComponentSchemaBuilderConfig,
+  buildModuleSchema: (
+    hasteModuleName: string,
+    ast: $FlowFixMe,
+    tryParse: ParserErrorCapturer,
+  ) => NativeModuleSchema,
+): SchemaType {
+  // Early return for non-Spec JavaScript files
+  if (
+    !contents.includes('codegenNativeComponent') &&
+    !contents.includes('TurboModule')
+  ) {
+    return {modules: {}};
+  }
```

Comment:
Can we add a couple of tests? Or at least one to tests that when there is no `codegenNativeComponent` and no `TurboModule`, it returns an emplty module.

Reply:
Yes I will test the function when i am unblocked